src/NuGetForUnity/Editor/NugetPreferences.cs

```diff
@@ -82,19 +87,20 @@ public override void OnGUI(string searchContext)
                     {
                         var pathCheck = NugetHelper.GetAssetsRelativePath(newPath);
 
-                        // if the path root is different or it is not under Assets folder, we issue a warning
-                        if (pathCheck == newPath || pathCheck.StartsWith(".."))
-                        {
-                            Debug.LogWarning(
-                                "Putting packages.config outside of Assets folder disables the functionality of restoring packages if the file is changed on the disk.");
-                        }
+                        // if the path root is different or it is not under Assets folder, we want to show a warning message
+                        shouldShowPackagesConfigPathWarning = pathCheck == newPath || pathCheck.StartsWith("..");
```

Comment:
```suggestion
                        shouldShowPackagesConfigPathWarning = pathCheck.StartsWith("..");
```
I would always show the warning if the path is outside the assets folder